DEVELOPMENTLOG.md

```diff
@@ -1,5 +1,73 @@
 # OpenLoco version 23.06.1+ (???)
 
+## Use height instead of width to initialize the RT buffer (#2001)
+
+Recent work on using a hardware backend for window scaling accidentally introduced a bug which could lead to
+failure to start OpenLoco when using window scaling. This rendered the game unplayable for some. This has now
+been fixed. Our apologies!
+
+## Allow scrolling up and down on steppers to change their values (#2019)
+
+One of the often-cited favourite quality of life features introduced by OpenRCT2, our sister project, is the
+ability to manipulate +/- stepper values using the mouse scrollwheel. @Broxzier's work has now been introduced
+in OpenLoco as well.
+
+## Fix Breakpad crash dump creation (#2032)
+
+For a few years, we have used the Breakpad library on Windows. This allows us to handle crashes gracefully,
+generating crash reports that may be analysed by us developers to find the cause. Unfortunately, a recent refactor
+of our CMake configuration files led to it being disabled for Windows builds as well. @ZehMatt has fixed this,
+so crashes are handled through Breakpad once more.
+
+## Introduce getClampedRange and getWorldRange (#2024)
+
+As you might image, the game often iterates over map tiles to perform certain calculations over them. This could
+be as a result of player interaction, e.g. applying a terraform tool, or without player interaction, e.g. calculating cargo generated by an industry. Either way, it is important that the range of tiles is both complete and valid. We have now introduced two helper functions that make sure our tile ranges do not extend outside the map bounds. This fixes a bug that some players reported when trying to change tile textures on the map edge.
```

Comment:
line length is a bit long here? seems like an accident since all other lines wrap nicely

Reply:
Not really an issue for MarkDown, but definitely unintentional. I'll amend.